src/intl/getFallbackLocale.ts

```diff
@@ -0,0 +1,26 @@
+import type { LangCode } from './locale';
+import { LocaleObject } from './localeObject';
+
+/**
+ * Fallback locales for each language.
+ * For example, a story has no en(global) locale, then we can take some fallback locale from here.
+ */
+export const localeFallback: Record<LangCode, LocaleObject[]> = {
+    en: [LocaleObject.fromAnyCode('en-US'), LocaleObject.fromAnyCode('en-GB')],
+    fr: [LocaleObject.fromAnyCode('fr-FR'), LocaleObject.fromAnyCode('fr-CA')],
+};
```

Comment:
I don't get why this is needed. You can use `LocaleObject` methods to find a suiting locale by language code (the part before the dash)

Reply:
it is not a locale code map, it is fallback priority, for example if a story doesn't have global `en` translation but have some other `en-*` locales, here is the priority what should be used for region-less `hrefLang` attribute

Reply:
If that's the case, we'd need much more languages covered than just English and French.
It's also worth making the comments and/or the variable naming a bit more clear about its intended usage (determining priority fallback languages)